pure_changelog.md

```diff
@@ -17,6 +17,7 @@
 * Changed responses in contacts and messages endpoints used to communicate with Center
 * Added SMS template call rejection window for case when no templates are defined
 * Changed unknown character glyph to the one matching the design
+* Changed alarm sounds and ringtones preview behavior to match the design
```

Comment:
it doesn't give any value to the end-user :P maybe it's possible to describe briefly how it should work?

Reply:
What would you say about something like `Changed alarm sounds and ringtones preview behavior - navigating through the list automatically switches playback to the currently selected one`? Isn't it too long as for the changelog entry?

Reply:
I'd say
`Fixed navigating through the ringtone preview list automatically switches playback to the currently selected one` or something like that

Reply:
Thanks, fixed!